src/InvalidConditionException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace ScriptFUSION\Mapper;
+
+/**
+ * The exception that is thrown when an invalid return value from a callback is specified.
```

Comment:
This exception isn't really concerned with return values. Let's write, *The exception that is thrown when an invalid condition is specified.*, instead.